### DISCUSSION TITLE
Any suggestions what kind of hyperparams should I check for NER model?

### DISCUSSION THREAD

**Discussion by XBeg9** [explosion/spaCy#10105] [Help: Best practices]
I am gonna run a large sweep for the NER model, any suggestions on what kind of params I should put in the list for the Wandb sweep run?

Thanks in advance.

**Comment by ljvmiranda921** [comment]
If you want to sweep through hyperparameters, you can probably start with different learning rates, schedulers, and optimizers. Another hyperparameter that might give you actionable results is dropout. 

In general, improving model performance will depend on the specifics of your problem/task (e.g. is there a class that's underperforming, do you have imbalanced data, what your docs look like).

**Reply by XBeg9** [reply]
Thanks! @ljvmiranda921 I have 2 labels inside the NER task, and the model has performed really poorly on one type of label (recall low, ~40%, compared to 75% vs the second type of label). I just did `dropout` and `learning_rate` optimisation and boosted model by 10%, was interesting what else I can do there. Anyway, thanks!
